code/modules/games/saintsandsins_cards.dm

```diff
@@ -0,0 +1,673 @@
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/rosemary_and_thyme
+	name = "Rosemary and Thyme"
+	ability = "Purges any damage-over-time spells on caster."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 6
+	)
+
+/decl/sas_card/spell/culling_of_the_firstborn
+	name = "Culling of the Firstborn"
+	ability = "Caster shares spell damage with casting Heretic."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/blood_boil
+	name = "Blood Boil"
+	ability = "Deals 1 Decay and 2 Fire damage to Heretic."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 7
+	)
+
+/decl/sas_card/spell/transmogrification
+	name = "Transmogrification"
+	ability = "Deals 2 damage of caster's chosen damage type."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_ALCHEMIST = 6
+	)
+
+///Fatebringer Kit
+/decl/sas_card/spell/star_crossed
+	name = "Star-crossed"
+	ability = "Deal 1 Holy damage to target for two rounds."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 8
+	)
+
+/decl/sas_card/spell/divine_fire
+	name = "Divine Fire"
+	ability = "Deal 1 Holy damage and 1 Fire damage to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 8
+	)
+
+/decl/sas_card/spell/cast_down
+	name = "Cast Down"
+	ability = "Nullifies spell and returns it to that caster's hand."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 6
+	)
+
+/decl/sas_card/spell/blood_tithe
+	name = "Blood Tithe"
+	ability = "Roll 1d4. Deal that much Water damage to ALL Heretics, including caster."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 6
+	)
+
+/decl/sas_card/spell/dire_portent
+	name = "Dire Portent"
+	ability = "Deal 4 Earth damage to target after 3 rounds."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 4
+	)
+
+/decl/sas_card/spell/hatred
+	name = "Hatred"
+	ability = "Share damage received with another Heretic."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 7
+	)
+
+/decl/sas_card/spell/karmic_balance
+	name = "Karmic Balance"
+	ability = "Deal 2 damage of last received damaged type to target."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 7
+	)
+
+/decl/sas_card/spell/unyielding_faith
+	name = "Unyielding Faith"
+	ability = "Convert spell's damage type to Holy."
+	spell_type = REACTION_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 7
+	)
+
+/decl/sas_card/spell/unnatural_bonds
+	name = "Unnatural Bonds"
+	ability = "Heal caster and one other Heretic by 3."
+	spell_type = KEY_TYPE
+	pack_probability = list(
+		CARDPACK_FATEBRINGER = 6
+	)
+
+/decl/sas_card/spell/suffering_poppets
+	name = "Suffering Poppets"
+	ability = "Reflects spell to all other Heretics."
```

Comment:
Dumb q: what happens if you reflect a spell targeting all Heretics? Do the others receive the effects twofold?

Reply:
Yes :)